packages/core/test/cacheManagerRedis.test.ts

```diff
@@ -46,15 +43,36 @@ describe("quit tests", () => {
 
   it("returns false when quit isn't OK", async () => {
     cacheManager.keyv = {
-      opts: { store: { redis: { quit: async () => "other thing" } } }
+      opts: {
+        store: { redis: { quit: async (): Promise<string> => "other thing" } }
+      }
     };
     return expect(cacheManager.quit()).to.eventually.be.false;
   });
 
   it("returns true when called with working redis", async () => {
     cacheManager.keyv = {
-      opts: { store: { redis: { quit: async () => "OK" } } }
+      opts: { store: { redis: { quit: async (): Promise<string> => "OK" } } }
     };
     return expect(cacheManager.quit()).to.eventually.be.true;
   });
 });
+
+describe("Redis connections", () => {
```

Comment:
Add test creating cache manager without connection and clusterConfig..

Reply:
Added tests for CacheManagerKeyv as the parent class.